lua/weapons/weapon_Auto_Shotgun.lua

```diff
@@ -0,0 +1,135 @@
+-- Easy Settings part
+
+-- SWEP Info
+SWEP.Author = "yousifgaming"
+SWEP.Instructions = "Primary is the same as Secondary fire automatic shotgun"
+SWEP.Purpose = "To play with Dingus"
+SWEP.Contact = "Don't Contact me i dont like people"
+
+-- SWEP info in spawnmenu
+SWEP.PrintName = "Auto Shotgun"
+SWEP.Category = "yousifgaming's Weapons"
+SWEP.Spawnable = true
+SWEP.AdminOnly = false
+SWEP.IconOverride = "materials/cat.jpg"
+
+if CLIENT then
+    SWEP.WepSelectIcon = surface.GetTextureID('cat')
+end
+
+-- Primary settings
+SWEP.Primary.ClipSize = 30
+SWEP.Primary.DefaultClip = 30
+SWEP.Primary.Automatic = true
+SWEP.Primary.Ammo = "Buckshot"
+
+-- Secondary settings
+SWEP.Secondary.Ammo = "None"
+SWEP.Secondary.Automatic = true
+
+-- Reload status
+SWEP.ReloadSound = "weapons/shotgun/shotgun_reload" .. math.random(1, 3) .. ".wav"

```

Comment:
Probably not what you intended, this is going to be random on each server boot (technically each swep init) rather than each reload.

Reply:
do i need to put it in the reload function?

Reply:
Yep, but again looks like you fixed it.